src/Workspaces/SharedUtilitiesAndExtensions/Workspace/CSharp/Extensions/ITypeSymbolExtensions.TypeSyntaxGeneratorVisitor.cs

```diff
@@ -70,6 +70,7 @@ public override TypeSyntax VisitArrayType(IArrayTypeSymbol symbol)
                 {
                     underlyingType = innerArray.ElementType;
 
+#if !CODE_STYLE // TODO: Remove the #if once NullableAnnotation is available.
```

Comment:
NullableAnnotation will not be available. This needs to use light-up.

Reply:
https://github.com/dotnet/roslyn/issues/41462 tracks reverting these conditional directives. This is likely going to be an issue even in future as new compiler APIs get added which are immediately available in IDE but not in CodeStyle layer. Let us discuss whether this should be tackled using reflection or bumping up the code style's referenced MS.CA version in the design meeting.